.github/ISSUE_TEMPLATE/summit-topic-proposal.md

```diff
@@ -0,0 +1,22 @@
+---
+name: "\U+270F Summit Topic Proposal"
+about: Submit a topic proposal for a session at the Collab Summit
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+Thank you! You are submitting a topic for the next Collaborator's Summit, Berlin 2019!
+
+Please include as much detail as you are able to at this moment. Don't worry, it doesn't have to be complete.
+
+Please feel free to link to any other issue, PR, resource that could be relevant.
```

Comment:
I think we usually put this kind of information into a comment that won't show up in the issue. [Here is an example.](https://github.com/nodejs/node/blob/master/.github/ISSUE_TEMPLATE/1-bug-report.md)

Reply:
```suggestion
Please feel free to link to any other issue, PR, or resource that could be relevant.
```

Reply:
Thanks for the suggestion @tniessen!

And for the suggested change @Trott!